docs/pages/tutorial/deploy-to-mainnet/main.md

```diff
@@ -0,0 +1,172 @@
+## Sepolia / Mainnet Deployment
+
+This is a guide to deploy a Dojo world to Sepolia.
+The steps for Mainnet are exactly the same, just replace the chain name and ID when necessary.
+
+
+### Setup
+
+* You need a [Starknet RPC Provider](https://www.starknet.io/fullnodes-rpc-services/) to deploy contracts on-chain. After you get yours, check if it works and is on the chain you want to deploy (`SN_SEPOLIA` or `SN_MAIN`)
+
+```sh
+# run this...
+curl --location '<RPC_PROVIDER_URL>' \
+--header 'Content-Type: application/json' \
+--data '{"id": 0,"jsonrpc": "2.0","method": "starknet_chainId","params": {}}'
+
+# you should get an output like this...
+{"jsonrpc":"2.0","result":"0x534e5f5345504f4c4941","id":0}
+
+# now paste the hex result part on this command... 
+echo 0x534e5f5345504f4c4941 | xxd -r -p
+
+# which !must! output SN_SEPOLIA or SN_MAIN
+SN_SEPOLIA
+```
+
+* Declare the `sepolia` profile in [`Scarb.toml`](https://github.com/rsodre/512karat/blob/main/dojo/Scarb.toml)
+
+```toml
+[profile.sepolia]
+```
+
+* Create the [`dojo_sepolia.toml`](https://github.com/rsodre/512karat/blob/main/dojo/dojo_sepolia.toml) dojo config file, with the same contents of [`dojo_dev.toml`](https://github.com/rsodre/512karat/blob/main/dojo/dojo_dev.toml), except for `[env]`, in which we're going to expose the `world_address` only:
+
+```toml
+[env]
+# rpc_url = ""         # env: STARKNET_RPC_URL
+# account_address = "" # env: DOJO_ACCOUNT_ADDRESS
+# private_key = ""     # env: DOJO_PRIVATE_KEY
+world_address = "0x30ce813d2c4d55298764d676bbc1f37fb8b6337e29337692c7133f52d106878"
+```
+
+* Clone the [`dev`](https://github.com/rsodre/512karat/blob/main/dojo/overlays/dev/) overlays to [`sepolia`](https://github.com/rsodre/512karat/blob/main/dojo/overlays/sepolia/)
+
+* Create `.env.sepolia` containing your RPC provider, account and private key. Make sure that account is deployed and has some [ETH](https://starknet-faucet.vercel.app) in it (0.001 is more than enough).
+
+```sh
+# usage: source .env.sepolia
+export STARKNET_RPC_URL=<RPC_PROVIDER_URL>
+export DOJO_ACCOUNT_ADDRESS=<YOUR_ACCOUNT_ADDRESS>
+export DOJO_PRIVATE_KEY=<YOUR_ACCOUNT_PRIVATE_KEY>
+```
+
+
+
+### Deployment
+
+* Load `.env.sepolia`.
+
+**IF FOR ANY REASON YOU ABORT THE DEPLOYMENT, JUMP TO THE CLEANUP STEP TO UNDO THIS**
+
+```sh
+cd dojo
+source .env.sepolia
+```
+
+* Build and check for errors. If the `migrate plan` command raises any errors, fix and try again before continuing.
+
+```sh
+cd dojo
+sozo -P sepolia clean
+sozo -P sepolia build
+sozo -P sepolia migrate plan
+```
+
+* Execute migration using the [`migrate`](https://github.com/rsodre/512karat/blob/main/dojo/migrate) script...
```

Comment:
This script is out to date and will not work with current dojo version.

Maybe it could also be simplified now, no more migrate plan, only migrate.

Reply:
Fixed => Set to just `migrate`

Maybe adding `sozo inspect` here too